app/views/purchases/index.html.erb

```diff
@@ -1,6 +1,9 @@
 <h1 class="title">Purchases</h1>
-<div class='purchase-index line-spacing'>
-  <h3 class="title is-4">Your last five purchases</h3>
+<div class='purchase-index line-spacing-top'>
+  <div class='purchase-header'>
+    <h3 class="title is-4">Last x purchases</h3>
+    <%= render 'buttons', :plimit => @plimit %>
```

Comment:
Do you need to pass in plimit here? The partial would have access to `@plimit` wouldn't it?